### PR TITLE
Update requests.yaml

### PR DESCRIPTION
**Pull request by MartaD**
Added app-operator and chart-operator to releases 13.0.0+ as per @rossf7 request. Removed duplicate request for containerLinux in versions >11.4.0 <13.0.0

### CODE REVIEW

**Review by whites11 — APPROVED** [review]

File: azure/requests.yaml [hunk 1]
```diff
@@ -34,3 +24,7 @@ releases:
           issue: https://github.com/giantswarm/giantswarm/issues/13671
         - name: kubernetes
           version: ">= 1.17.12"
+        - name: app-operator
+          version: ">=2.3.2"
+        - name: chart-operator
+          version: ">=2.3.3"
```
Comment (rossf7) [inline azure/requests.yaml:30]:
```suggestion
          version: ">=2.3.5"
```

Thanks Marta! Sorry to be a pain but could you include the chart-operator v2.3.5? It's a bug fix release we created yesterday.

https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#235---2020-10-13

Comment (MartaD) [reply azure/requests.yaml:30]: sure, we're not testing the release yet anyway

**Review by rossf7 — APPROVED** [review]
Thanks, LGTM

File: azure/requests.yaml [hunk 2]
```diff
@@ -34,3 +24,7 @@ releases:
           issue: https://github.com/giantswarm/giantswarm/issues/13671
         - name: kubernetes
```
Comment (MartaD) [inline azure/requests.yaml:25]: @whites11 you wanted 1.17.12 for the alpha release right? is this request covering alpha as well? For >=13.0.0 we want 1.18.x. And if it covers alphas then this PR would create a mess..

File: azure/requests.yaml [hunk 3]
```diff
@@ -34,3 +24,7 @@ releases:
...
           version: ">= 1.17.12"
```
Comment (MartaD) [inline azure/requests.yaml:26]: @whites11 you wanted 1.17.12 for the alpha release right? is this request covering alpha as well? For >=13.0.0 we want 1.18.x. And if it covers alphas then this PR would create a mess..

**Pull request closed without merging**